K2Bridge/Visitors/RangeClauseVisitor.cs

```diff
@@ -29,8 +30,19 @@ public void Visit(RangeClause rangeClause)
             {
                 // general "is between" filter on numeric fields uses a rangeClause query with GTE+LT (not LTE like above)
                 EnsureClause.IsNotNull(rangeClause.LTValue, nameof(rangeClause.LTValue));
-
-                rangeClause.KustoQL = $"{rangeClause.FieldName} >= {rangeClause.GTEValue} and {rangeClause.FieldName} < {rangeClause.LTValue}";
+                var t = ClauseFieldTypeProcessor.GetType(schemaRetriever, rangeClause.FieldName).Result;
+                switch (t)
+                {
+                    case ClauseFieldType.Numeric:
+                    case ClauseFieldType.Text:
+                        rangeClause.KustoQL = $"{rangeClause.FieldName} >= {rangeClause.GTEValue} and {rangeClause.FieldName} < {rangeClause.LTValue}";
+                        break;
```

Comment:
1. Range clause can't work with text types in ADX. this case should throw an error.
2. Something is wrong in the tests since a query like: `field >= aaa and field < bbb` should fail since you're missing quotes

Reply:
fixed